ansible/create-aws-instance.yml

```diff
@@ -0,0 +1,15 @@
+---
+#
```

Comment:
Could we add a one line description of the purpose of this playbook something along the lines of "Creates AWS instances for hosting Kind or k3d clusters"